lib/headers.js

```diff
@@ -30,11 +30,12 @@ module.exports = function (app, options) {
   ]);
 
   //extend rest body parser to also parse application/vnd.api+json
-  remotes.options.json = {
+  remotes.options.json = remotes.options.json || {};
+  remotes.options.json = _.extend(remotes.options.json, {
```

Comment:
scratch that, reading the docs again, i think you are right to use extends/assign rather than defaults as you want to be able to overwrite the defaults